model/src/test/utils.py

```diff
@@ -6,37 +6,64 @@
 
 import numpyro.distributions as dist
 import pyrenew.transformation as t
-from pyrenew.metaclass import DistributionalRV, TransformedRandomVariable
-from pyrenew.process import SimpleRandomWalkProcess
+from pyrenew.metaclass import (
+    DistributionalRV,
+    RandomVariable,
+    SampledValue,
+    TransformedRandomVariable,
+)
+from pyrenew.process import RandomWalk
 
 
-def simple_rt(arg_name: str = "Rt_rv"):
+class SimpleRt(RandomVariable):
     """
-    Helper function to create a default Rt
+    Helper class to create a default Rt
     RandomVariable for testing.
-
-    Parameters
-    -----------
-    arg_name : str
-        Name assigned to the randonvariable.
-        If None, then defaults to "Rt_rv"
-
-    Returns
-    -------
-    TransformedRandomVariable :
-       A log-scale random walk with fixed
-       init value and step size priors
     """
-    return TransformedRandomVariable(
-        arg_name,
-        base_rv=SimpleRandomWalkProcess(
-            name="log_rt",
-            step_rv=DistributionalRV(
-                name="rw_step_rv", distribution=dist.Normal(0, 0.025)
-            ),
-            init_rv=DistributionalRV(
-                name="init_log_rt", distribution=dist.Normal(0, 0.2)
+
+    def __init__(self, name: str = "Rt_rv"):
+        """
+        Default constructor
+
+        Parameters
+        -----------
+        name : str
+           Name assigned to the RandomVariable.
+           If None, then defaults to "Rt_rv"
+
+        Returns
+        -------
+        None
+        """
+        self.name = name
+        self.rt_rv_ = TransformedRandomVariable(
+            name=name + "log_rt_random_walk",
```

Comment:
```suggestion
            name=f"{name}_log_rt_random_walk",
```

Reply:
Actually, this seems like an inappropriate name for the transformed random variable. I think we should drop the `log` part.